examples/hiop/ex9p.cpp

```diff
@@ -96,6 +96,7 @@ class LinearScaleOperator : public Operator
    {
       Vector w_glob(width);
       pfes.Dof_TrueDof_Matrix()->MultTranspose(w, w_glob);
+      w_glob.HostReadWrite(); // read+write -> can use w_glob(i) (non-const)
       for (int i = 0; i < width; i++) { grad(0, i) = w_glob(i); }
```

Comment:
Not preferable to use `AsConst`?

Reply:
We can use `w_glob.HostRead()` and then `AsConst(w_glob)(i)` -- this will be better if the array is then needed on device. In this case, the array is deleted after the `for` loop, so `HostReadWrite` without `AsConst` does the same job.